src/atc/mount/main.py

```diff
@@ -2,10 +2,20 @@
 import json
 from types import SimpleNamespace
 
+import deprecation
+
+import atc
 from atc.atc_exceptions import AtcException
 from atc.functions import init_dbutils
 
 
+@deprecation.deprecated(
+    deprecated_in="1.0.48",
+    removed_in="2.0.0",
+    current_version=atc.__version__,
+    details="use direct access instead. "
+    "See the atc-dataplatform unittests, for example.",
+)
 def main():
     parser = argparse.ArgumentParser(description="atc-dataplatform mountpoint setup.")
     parser.add_argument(
```

Comment:
Should we make it possible for others to use this direct access main? So it is a function that can be called. (More generalized).